airbyte-cdk/python/airbyte_cdk/test/catalog_builder.py

```diff
@@ -1,29 +1,62 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 
-from typing import Any, Dict, List
+from typing import List, Union, overload
 
-from airbyte_protocol.models import ConfiguredAirbyteCatalog, SyncMode
+from airbyte_protocol.models import ConfiguredAirbyteCatalog, SyncMode, ConfiguredAirbyteStream
```

Comment:
I ended up not needed that change because the MessageGrouper can't rely on the catalog like I thought because the catalog passed is [a big dummy](https://github.com/airbytehq/airbyte-platform-internal/blob/b5851defe90adac15e8c6d8b117b645e65e7690e/oss/airbyte-connector-builder-server/src/main/java/io/airbyte/connector_builder/requester/AirbyteCdkRequesterImpl.java#L47-L60)